pulp_container/app/urls.py

```diff
@@ -27,9 +27,9 @@
 router.register(r"^v2/(?P<path>.+)/manifests", Manifests, basename="manifests")
 
 urlpatterns = [
-    url(r"^token/$", BearerTokenView.as_view()),
-    url(r"^v2/$", VersionView.as_view()),
-    url(r"^v2/_catalog", CatalogView.as_view()),
-    url(r"^v2/(?P<path>.+)/tags/list", TagsListView.as_view()),
-    url(r"", include(router.urls)),
+    re_path(r"^token/$", BearerTokenView.as_view()),
+    re_path(r"^v2/$", VersionView.as_view()),
+    path("v2/_catalog", CatalogView.as_view()),
+    re_path(r"^v2/(?P<path>.+)/tags/list", TagsListView.as_view()),
```

Comment:
I would have expected
```suggestion
    path("v2/<path>/tags/list", TagsListView.as_view()),
```
to work. But `re_path` is fine!

Reply:
This line does work both ways, however with the other two I'd need to stick to the ``re_path``.

Reply:
Actually, I am taking my words back.
```
In [3]: path=path("v2/<path>/tags/list", TagsListView.as_view())

In [4]: path.resolve('v2/library/busybox/tags/list')

In [5]: path.resolve('v2/busybox/tags/list')
Out[5]: ResolverMatch(func=pulp_container.app.registry_api.TagsListView, args=(), kwargs={'path': 'busybox'}, url_name=None, app_names=[], namespaces=[], route=v2/<path>/tags/list)

In [6]: x=re_path(r"^v2/(?P<path>.+)/tags/list", TagsListView.as_view())

In [7]: x.resolve('v2/library/busybox/tags/list')
Out[7]: ResolverMatch(func=pulp_container.app.registry_api.TagsListView, args=(), kwargs={'path': 'library/busybox'}, url_name=None, app_names=[], namespaces=[], route=^v2/(?P<path>.+)/tags/list)

In [8]: x.resolve('v2/busybox/tags/list')
Out[8]: ResolverMatch(func=pulp_container.app.registry_api.TagsListView, args=(), kwargs={'path': 'busybox'}, url_name=None, app_names=[], namespaces=[], route=^v2/(?P<path>.+)/tags/list)

```

Reply:
Thanks for testing!

Reply:
Digging more into the docs: https://docs.djangoproject.com/en/3.2/topics/http/urls/#path-converters
`path=path("v2/<path:path>/tags/list", TagsListView.as_view())` may work, but my approval stands either way.

Reply:
yes, it is the syntax i did not get properly! I was reading the docs plus checked the code and was really getting confused why it does not work! https://github.com/django/django/blob/main/django/urls/converters.py#L39
Thanks!!

Reply:
here we go:
```In [53]: x=path("v2/<path:path>/tags/list", TagsListView.as_view())

In [54]: x.pattern.match('v2/busybox/tags/list')
Out[54]: ('', (), {'path': 'busybox'})

In [55]: x.pattern.match('v2/library/busybox/tags/list')
Out[55]: ('', (), {'path': 'library/busybox'})
```